### DISCUSSION TITLE
CBC Sport Azerbaijan

### DISCUSSION THREAD

**Discussion by pukozs75** [iptv-org/community#777] [Q&A]
### Channel Name

CBC Sport

### Channel ID

CBCSport.az

### Website

http://www.cbcsport.az/live

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
http://s69362.cdn.ngenix.net/live/_definst_/adaptive.smil/playlist.m3u8 geo

**Comment by freearhey** [comment]
http://s69362.cdn.ngenix.net/live/_definst_/adaptive.smil/playlist.m3u8 (offline)
<img width="488" alt="image" src="https://user-images.githubusercontent.com/7253922/232227110-ace65463-2be4-4e6c-883d-85dd650b45ce.png">

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19755
